client.go

```diff
@@ -31,6 +31,7 @@ type Client struct {
 	userAgent  string
 
 	IPs             *IPService
+	Loadbalancers   *LoadbalancerService
```

Comment:
Can we rename the `Loadbalancer` instances to `LoadBalancer` throughout the code?